src/Sylius/Component/Review/Calculator/AverageRatingCalculator.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Review\Calculator;
+
+use Sylius\Component\Review\Model\ReviewableInterface;
+use Sylius\Component\Review\Model\ReviewInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class AverageRatingCalculator implements AverageRatingCalculatorInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function calculate(ReviewableInterface $reviewable)
+    {
+        if (0 === count($reviews = $reviewable->getReviews())) {
```

Comment:
Can we put the assignment before the if? I find it really counterintuitive to assign variables values in control structures.